metecho/api/serializers.py

```diff
@@ -144,8 +164,8 @@ class ProjectSerializer(HashIdModelSerializer):
     repo_url = serializers.SerializerMethodField()
     repo_image_url = serializers.SerializerMethodField()
     has_push_permission = serializers.SerializerMethodField()
-    github_users = StringListField()
-    org_config_names = StringListField(read_only=True)
+    github_users = GitHubUserSerializer(many=True, allow_empty=True, required=False)
```

Comment:
@jerivas I just noticed this `required=False` -- is that intentional? From what I can tell, that's a change from the existing functionality.

Reply:
Before #1495 `github_users` was not declared as a explicit serializer field, so it followed the model field which is `JSONField(blank=True, default=list)`. This should be equivalent to that.

Now that I think of it, shouldn't it be read-only?

Reply:
Ah, I think that's true on both counts. 👍

Reply:
Oh, just went to https://github.com/oddbird/Metecho/pull/1376 and noticed we need this field to be writable because that determines the members of the team for a given Project. I think there's nothing else to do for now.